acceptance/repository-cloudant/docker.setup.js

```diff
@@ -0,0 +1,321 @@
+// Copyright IBM Corp. 2017,2019. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+const _ = require('lodash');
+const async = require('async');
+const docker = new require('dockerode')();
+const http = require('http');
+const ms = require('ms');
+const fs = require('fs-extra');
```

Comment:
`fs-extra` isn't listed in `package.json`.. but it also doesn't appear to really be fully utilized here since the only method used is in `fs` from core.

Reply:
true `fs` is enough